CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# v2.9.0 (2024-10-09)
+
+## OS Changes
+* Add nvidia-persistenced and nvidia-modprobe to kmod-*-nvidia ([#122])
+* Add NVIDIA time-slicing settings to nvidia-k8s-device-plugin ([#169])
+* libcap: fix cross-compile toolchain usage ([#174])
+* login: start the getty services earlier ([#175])
```

Comment:
Too late for this comment, but we should fix this later since these two lines don't match the wording/format we use in the CHANGELOG.

Reply:
Sure, I can wordsmith these and follow up